source/dub/generators/generator.d

```diff
@@ -999,6 +1000,7 @@ const(string[string])[] makeCommandEnvironmentVariables(CommandType type,
 	env["LIBS"]                  = join(build_settings.libs, " ");
 	env["SOURCE_FILES"]          = join(build_settings.sourceFiles, " ");
```

Comment:
also want to add cSourceFiles as environment variable?

Reply:
Good point. Added.